### ISSUE TITLE
H1BP_DUMP is silently skipped in graph mode — blocks packed-path debugging

### ISSUE THREAD

**Issue by bong-water-water-bong** [1bit-MONSTER/1bit-MONSTER#1626]
## Summary

`H1BP_DUMP` (the bit-perfect bisection dump in `forward_dev`) never fires when the Phase 2 graph path is active: `generate_fast` replays the captured graph (built with `with_dumps=false`), so no dump files are written and there is no warning.

This blocked diagnosing #1623 (TQ2NZ degenerate output) — the natural next step is dumping logits for the packed path, but graph mode silently bypasses the instrumentation.

## Repro

```
H1BP_DUMP=/tmp/dump ./build/bench_hip_1bp <tq2nz-model.1bp> 3 0
# "Phase 2: decode graph captured OK" — /tmp/dump stays empty, no warning
```

## Ask

- Warn when `H1BP_DUMP` is set but the graph path will skip it, or
- Add a `--no-graph` / `H1BP_GRAPH=0`-style env toggle so eager-mode instrumentation works on packed models (also useful for isolating #1624's graph-replay crash).

**Comment by bong-water-water-bong** [comment]
Fixed in commit 9735974d (local): H1BP_GRAPH=0 disables graph capture so H1BP_DUMP fires in eager mode; generate_fast honors H1BP_DUMP in eager mode. Verified: 896 dump files per 2-token run on both Q4NX and TQ2NZ models. Note: commits are local-only until pushed.
